recipes/anyrpc/all/conanfile.py

```diff
@@ -0,0 +1,124 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy, rm, rmdir
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+import os
+
+
+required_conan_version = ">=1.52.0"
+
+
+class AnyRPCConan(ConanFile):
+    name = "anyrpc"
+    description = "A multiprotocol remote procedure call system for C++"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/sgieseking/anyrpc"
+    topics = ("rpc")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_log4cplus": [True, False],
+        "with_threading": [True, False],
+        "with_regex": [True, False],
+        "with_wchar": [True, False],
+        "with_protocol_json": [True, False],
+        "with_protocol_xml": [True, False],
+        "with_protocol_messagepack": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_log4cplus": False,
+        "with_threading": True,
+        "with_wchar": True,
+        "with_regex": True,
+        "with_protocol_json": True,
+        "with_protocol_xml": True,
+        "with_protocol_messagepack": True,
+    }
+
+    @property
+    def _minimum_cpp_standard(self):
+        return 11
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        if self.options.with_log4cplus:
+            self.requires("log4cplus/2.0.7")
+
+    def validate(self):
+        if self.info.settings.compiler.cppstd:
+            check_min_cppstd(self, self._minimum_cpp_standard)
+
+        if self.options.with_log4cplus and self.options.with_wchar:
```

Comment:
Can you use `self.info.options` here?

```suggestion
        if self.info.options.with_log4cplus and self.info.options.with_wchar:
```

Reply:
done